packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -16,11 +16,6 @@
 	transition: border-color 0.1s linear, box-shadow 0.1s linear;
 	@include reduce-motion("transition");
 
-	// Allow overflow on desktop.
-	@include break-small() {
-		overflow: inherit;
```

Comment:
I wonder if this started causing issues because the component was nested under a different element. Strikes me that `inherit` should probably be avoided in the root of react component styles, because it add couples the component to its parent.

Reply:
Had to revert this as it causes lots of tests to fail, though I can't see why. I'm unable to reproduce the issues the tests are having.